letsencrypt/plugins/util_test.py

```diff
@@ -98,6 +98,11 @@ def test_listening_ipv6(self, mock_get_utility, mock_process, mock_net):
         self.assertEqual(mock_get_utility.call_count, 1)
         mock_process.assert_called_once_with(4420)
 
+    @mock.patch("letsencrypt.plugins.util.psutil.net_connections")
+    def test_access_denied_exception(self, mock_net):
+        mock_net.return_value = False
```

Comment:
Eh well, the exception logs and returns False, so I figured I'd cover all the bases.